inc/osvr/Client/DisplayConfig.h

```diff
@@ -80,6 +86,9 @@ namespace client {
                                              OSVR_SurfaceCount surface) const;
 
         OSVR_CLIENT_EXPORT bool isStartupComplete() const;
+        OSVR_CLIENT_EXPORT OSVR_DisplayInputCount getNumDisplayInputs() const;
+        OSVR_CLIENT_EXPORT const ViewerEye &
```

Comment:
I really am not a big fan of this method - it implies there's only one viewer eye per display input, which is usually not the case. Display inputs don't know their surfaces, but surfaces know their display inputs.